metadata-load/src/main/java/uk/ac/ebi/ampt2d/metadata/loader/core/xml/SraAnalysisXmlParser.java

```diff
@@ -0,0 +1,43 @@
+/*
+ *
+ * Copyright 2019 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package uk.ac.ebi.ampt2d.metadata.loader.core.xml;
+
+import org.apache.xmlbeans.XmlException;
+import uk.ac.ebi.ampt2d.metadata.loader.api.SraApiAnalysisRetriever;
+import uk.ac.ebi.ena.sra.xml.ANALYSISDocument;
+import uk.ac.ebi.ena.sra.xml.AnalysisType;
+
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public class SraAnalysisXmlParser extends SraXmlParser<AnalysisType> {
```

Comment:
I think we should have SraAnalysisDocumentXmlParser , because the xml from db uses AnalysisSetDocument.

Reply:
I used simply "analysis" to reflect the fact that what we are parsing represents an analysis, not to make it dependent on a particular class name from the ENA API (that could always change).

Reply:
yes, I agree.